src/engraving/dom/measure.cpp

```diff
@@ -763,7 +763,6 @@ void Measure::add(EngravingItem* e)
         while (s && s->rtick() == t) {
             if (!seg->isChordRestType() && (seg->segmentType() == s->segmentType())) {
                 LOGD("there is already a <%s> segment", seg->subTypeName());
-                return;
```

Comment:
Does this mean that we're allowing to insert two segments of the same type at the same tick? If so, I think it's fine for now, but let's keep in mind that we're just moving the problem from one place to another, meaning we now keep the undo/redo stack in sync, but in turn we're creating redundant information in the score which could cause problems elsewhere. For now I agree to do this, but let's leave a big comment here referencing to this issue, cause we'll need to come back to this :)

Reply:
Yes it does - I'll leave a comment.  I'm working on much larger changes to `ScoreRange` (which handles join measures, split measures and `rewriteMeasures` logic) for 4.5.  I believe this will remove the problematic code which causes this to happen, but will obviously check and restore this line asap if possible.